lib/widgets/video_viewer_mobile_theme.dart

```diff
@@ -28,7 +27,7 @@ class VideoViewerMobileTheme extends StatelessWidget {
           TwakeIconButton(
             tooltip: L10n.of(context)!.back,
             icon: Icons.close,
-            onTap: () => context.pop(),
+            onTap: () => Navigator.of(context).pop(),
```

Comment:
Why use navigator and not go_router? What's different here?

Reply:
use `context.pop` will result the screen back to chat list screen, because the dialog is pushed using Navigator, so...